account_partner_reconcile/README.rst

```diff
@@ -0,0 +1,88 @@
+=========================
+Account Partner Reconcile
+=========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--reconcile-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-reconcile/tree/13.0/account_partner_reconcile
```

Comment:
```suggestion
    :target: https://github.com/OCA/account-reconcile/tree/14.0/account_partner_reconcile
```
shouldn't be changed?

Reply:
This file will be regenerated automatically by robot: then I didn't changed it.